### ISSUE TITLE
[REQUESTED CONTENT] Fetch and Display Chart Data using Chart.js and Webiny Headless CMS

### ISSUE THREAD

**Issue by endymion1818** [webiny/write-with-webiny#49]
### Introduction

Whether it's for an organizations online Annual Report, or to display results from some campaign, building charts is one of the staples of JavaScript development. But where do you store the data? What if there's a reassessment of the results when you're in the middle of building it? Rather than hard code your data, you can use a Headless CMS to store that data. With it's flexible content modeling and role-based access, Webiny Headless CMS is a great choice to store data for your chart so that it's independent of your codebase.

### Goals

In this tutorial we will build three different charts with data sourced from Webiny Headless CMS.

### Outline

  - What is a Headless CMS?
  - Why use Webiny CMS?
  - Setting up Webiny CMS
    -- Content Structure
  - Creating, Updating, and Deleting Chart Data
  - Building a Chart application
    -- Installing dependencies
    -- Connecting the CMS to the application
    -- Creating a Doughnut Chart
    -- Creating a Bar Chart
    -- Creating a Line Graph
  - Conclusion
  - Resources

**Comment by sid-khuntwal** [comment]
Would like to work on this one @endymion1818

**Comment by endymion1818** [comment]
Hi @sid-khuntwal, that's great! Have you worked with a charting library before? Could we also see some samples of your writing on a similar technical topic?

**Comment by sid-khuntwal** [comment]
Yes, I have worked on chart.js. for my writing samples, I would request you to have a look at my blog - sidlyf.com

**Comment by endymion1818** [comment]
Hey @sid-khuntwal, we got your submission and have written a few comments for you ... do you think you can complete the article soon?

**Comment by sid-khuntwal** [comment]
Hey so sorry to be missing on this, I met with an accident recently and am currently in the hospital. I don't think I would be able to review the changes anytime soon

**Comment by endymion1818** [comment]
Oh no! So sorry to hear that @sid-khuntwal, no worries we'll pause your article, please let us know when you're able to pick it up again.

**Comment by endymion1818** [comment]
@sid-khuntwal I'm assigning this back out again, I hope you have recovered, please contact me via the Slack channel if you would like to pick up another article.

**Comment by EmmanuelTheCoder** [comment]
Can I take this @endymion1818 ?

**Comment by menard-codes** [comment]
This seems interesting. Would like to request if I can write this tutorial, @swapnilmmane ?

**Comment by Marienoir** [comment]
Hello @endymion1818 
I will like to write content for this article topic
Looking forward to hearing from you

**Comment by Nomzy-kush** [comment]
Hello @endymion1818, I will like to take this topic on. I believe I have the qualifications.  I'd be happy to send you a detailed draft for review. Thank you for considering my pitch, and I look forward to hearing back from you soon.
